src/data_structures.rs

```diff
@@ -357,3 +365,23 @@ impl<F: Field> core::ops::Deref for LinearCombination<F> {
         &self.terms
     }
 }
+
+/// Helper macro to forward all derived implementations to the ToBytes and FromBytes traits
+#[macro_export]
+macro_rules! delegate {
```

Comment:
`delegate` is a bit unintuitive, perhaps `derive_serializers`, `derive_bytes`, or similar would be more clear

Reply:
`derive_bytes` sounds good